Jenkinsfile

```diff
@@ -66,20 +66,25 @@ def job = {
         downstreamBuildsStepName: {
             echo "Building cp-downstream-builds"
             if (config.isPrJob) {
-                def muckrakeBranch = kafkaMuckrakeVersionMap[env.CHANGE_TARGET]
-                def forkRepo = "${env.CHANGE_FORK ?: "confluentinc"}/kafka.git"
-                def forkBranch = env.CHANGE_BRANCH
-                echo "Schedule test-cp-downstream-builds with :"
-                echo "Muckrake branch : ${muckrakeBranch}"
-                echo "PR fork repo : ${forkRepo}"
-                echo "PR fork branch : ${forkBranch}"
-                buildResult = build job: 'test-cp-downstream-builds', parameters: [
-                        [$class: 'StringParameterValue', name: 'BRANCH', value: muckrakeBranch],
-                        [$class: 'StringParameterValue', name: 'TEST_PATH', value: "muckrake/tests/dummy_test.py"],
-                        [$class: 'StringParameterValue', name: 'KAFKA_REPO', value: forkRepo],
-                        [$class: 'StringParameterValue', name: 'KAFKA_BRANCH', value: forkBranch]],
-                        propagate: true, wait: true
-                return ", downstream build result: " + buildResult.getResult();
+                try {
+                    def muckrakeBranch = kafkaMuckrakeVersionMap[env.CHANGE_TARGET]
+                    def forkRepo = "${env.CHANGE_FORK ?: "confluentinc"}/kafka.git"
+                    def forkBranch = env.CHANGE_BRANCH
+                    echo "Schedule test-cp-downstream-builds with :"
+                    echo "Muckrake branch : ${muckrakeBranch}"
+                    echo "PR fork repo : ${forkRepo}"
+                    echo "PR fork branch : ${forkBranch}"
+                    buildResult = build job: 'test-cp-downstream-builds', parameters: [
+                            [$class: 'StringParameterValue', name: 'BRANCH', value: muckrakeBranch],
+                            [$class: 'StringParameterValue', name: 'TEST_PATH', value: "muckrake/tests/dummy_test.py"],
+                            [$class: 'StringParameterValue', name: 'KAFKA_REPO', value: forkRepo],
+                            [$class: 'StringParameterValue', name: 'KAFKA_BRANCH', value: forkBranch]],
+                            propagate: true, wait: true
+                    return "cp-downstream-builds SUCCESS";
+                } catch (ignored) {
+                    currentBuild.result = 'UNSTABLE'
+                    return "cp-downstream-builds FAILURE";
```

Comment:
Do we not include the failing build number anymore?

Reply:
Was that ever due to the string returned here? I have never seen the `, downstream build result:` text in the jenkins console output. I think it may be due to the fact that we're catching it the failure, rather than the text we return. I'm not sure how to best deal with this yet. We do print the job number near the top but it would be better to have it at the end as before.

Reply:
I was trying to understand how this changes things. As you say, we previously had the number at the end. Is that gone now?

Reply:
Yes it is and thanks for noticing. I'm following up with others with how we can possibly get it back. It is printed earlier in the run but it's harder to notice it there.

Reply:
I've changed it to store the result in a variable and then echo it at the end. This should make it pretty close to the old behavior.